lib/topological_inventory/ansible_tower/collector.rb

```diff
@@ -86,8 +88,9 @@ def collector_thread(connection, entity_type)
       # connection.api.jobs.all
       # connection.api.workflow_jobs.all
     rescue => e
+      metrics.record_error
       logger.error("Error collecting :#{entity_type}, message => #{e.message}")
-      raise e
+      logger.error(e)
```

Comment:
Why are you making this change here?

Reply:
because I want to unify behaviour with other collectors